tests/utils/dockercli/volumelifecycle.go

```diff
@@ -51,6 +57,26 @@ func AttachVolumeWithRestart(ip, volName, containerName string) (string, error)
 		dockercli.TestContainer)
 }
 
+// WriteToVolume write data to a given file on given volume
+func WriteToVolume(ip, volName, containerName, fileName, data string) (string, error) {
+	log.Printf("Writing %s to file %s on volume [%s] on VM[%s]\n", data, fileName, volName, ip)
+
+	writeCmd := " /bin/sh -c 'echo \"" + data + "\" > /vol1/test.txt'"
+	return ssh.InvokeCommand(ip, dockercli.RunContainer+" -v "+volName+
+		":/vol1 --name "+containerName+dockercli.ContainerImage+
+		writeCmd)
+}
+
+// ReadFromVolume read content of given file on a given volume
+func ReadFromVolume(ip, volName, containerName, fileName string) (string, error) {
+	log.Printf("Reading from file %s on volume [%s] on VM[%s]\n", fileName, volName, ip)
+
+	readCmd := " /bin/sh -c 'cat /vol1/" + fileName + "'"
+	return ssh.InvokeCommand(ip, dockercli.RunContainer+" -v "+volName+
```

Comment:
dockercli.RunContainer doesn;t include the "-rm" flag as it should, so, this container will remain in the system and won;t be removed in the teardown function. Please update dockercli.RunContainer to include "-rm".